skbio/stats/ordination/__init__.py

```diff
@@ -113,12 +113,13 @@
 from ._correspondence_analysis import CA
 from ._redundancy_analysis import RDA
 from ._canonical_correspondence_analysis import CCA
-from ._principal_coordinate_analysis import PCoA
+from ._principal_coordinate_analysis import PCoA, PCoABase
```

Comment:
It is necessary for our benchmarking as were adding in a derived class. Can
import direct but that is not assured to be stable
On Mar 25, 2015 12:07 PM, "Evan Bolyen" <notifications@github.com> wrote:

> In skbio/stats/ordination/__init__.py
> <https://github.com/biocore/scikit-bio/pull/879#discussion_r27148722>:
>
> > @@ -113,12 +113,13 @@
> >  from ._correspondence_analysis import CA
> >  from ._redundancy_analysis import RDA
> >  from ._canonical_correspondence_analysis import CCA
> > -from ._principal_coordinate_analysis import PCoA
> > +from ._principal_coordinate_analysis import PCoA, PCoABase
>
> [image: :+1:] Base classes probably don't make sense to export generally,
> unless you would expect users to extend said base classes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/879/files#r27148722>.
>